.cargo/config.toml

```diff
@@ -0,0 +1,22 @@
+# General rustflags for all non-wasm32 targets
+[target.'cfg(not(target_arch = "wasm32"))']
+rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
+
+[profile.release.target.wasm32-unknown-unknown]
+debug = true
+
+[source.crates-io]
+replace-with = "vendored-sources"
+
+[source."git+https://github.com/o1-labs/proof-systems?rev=0c64141221529455d1dfcd72570c9f40834c1280"]
+git = "https://github.com/o1-labs/proof-systems"
+rev = "0c64141221529455d1dfcd72570c9f40834c1280"
+replace-with = "vendored-sources"
+
+[source.vendored-sources]
+directory = "src/lib/crypto/mina-rust-dependencies/"
```

Comment:
Optional: I'd put it to the top of the file. I got a bit confused at the absense of `./vendored-sources/` subdirectory.